### PR TITLE
[SPARK-35345][SQL] Add Parquet tests to BloomFilterBenchmark

### PR DESCRIPTION
**Pull request by huaxingao**
### What changes were proposed in this pull request?
Add BloomFilter Benchmark test for Parquet

### Why are the changes needed?
Currently, we only have BloomFilter Benchmark test for ORC. Will add one for Parquet too.

### Does this PR introduce _any_ user-facing change?
no

### How was this patch tested?
new test

### CODE REVIEW

File: sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala [hunk 1]
```diff
@@ -81,8 +80,55 @@ object BloomFilterBenchmark extends SqlBasedBenchmark {
...
+    }
+  }
+
   override def runBenchmarkSuite(mainArgs: Array[String]): Unit = {
-    writeBenchmark()
-    readBenchmark()
+    writeORCBenchmark()
+    readORCBenchmark()
+    writeParquetBenchmark()
+    readParquetBenchmark()
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala:132]: maybe we should better update the results file together for JDK 8, 11 and 17.

Comment (huaxingao) [reply sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala:132]: Will update the results file once Git Actions benchmark tests finish.

**Review by HyukjinKwon — APPROVED** [review]

File: sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala [hunk 2]
```diff
@@ -81,8 +80,55 @@ object BloomFilterBenchmark extends SqlBasedBenchmark {
...
+      12 * 1024 * 1024, 16 * 1024 * 1024, 32 * 1024 * 1024)
+    for (blocksize <- blockSizes) {
+      withTempPath { dir =>
+        val path = dir.getCanonicalPath
+        df.write.option("parquet.block.size", blocksize).parquet(path + "/withoutBF")
+        df.write.option(ParquetOutputFormat.BLOOM_FILTER_ENABLED + "#value", true)
+          .option("parquet.block.size", blocksize)
+          .parquet(path + "/withBF")
+
+        runBenchmark(s"Parquet Read") {
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala:113]:
```suggestion
        runBenchmark("Parquet Read") {
```

File: sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala [hunk 3]
```diff
@@ -81,8 +80,55 @@ object BloomFilterBenchmark extends SqlBasedBenchmark {
     }
   }
 
+  private def writeParquetBenchmark(): Unit = {
+    withTempPath { dir =>
+      val path = dir.getCanonicalPath
+
+      runBenchmark(s"Parquet Write") {
```
Comment (HyukjinKwon) [inline sql/core/src/test/scala/org/apache/spark/sql/execution/benchmark/BloomFilterBenchmark.scala:87]:
```suggestion
      runBenchmark("Parquet Write") {
```

**Comment by HyukjinKwon** [comment]
Merged to master.

**Comment by huaxingao** [comment]
Thank you Hyukjin!

**Pull request closed without merging**